src/components/Education.js

```diff
@@ -1,34 +1,131 @@
 import styles from './education.module.css';
+import { useState, Fragment } from 'react';
+import { format } from 'date-fns';
 
-function handleClick(e) {}
+export default function Education({ data, onChange, onClick }) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  function handleClick() {
+    isEditing ? setIsEditing(false) : setIsEditing(true);
+  }
+
+  function generateContent(schoolData) {
+    let content = (
+      <div className={styles.infoContainer}>
+        <label htmlFor="school">
+          School
+          <div className="school" id="school">
+            {schoolData.school}
+          </div>
+        </label>
+        <label htmlFor="title">
+          Title of studies
+          <div className="title" id="title">
+            {schoolData.title}
+          </div>
+        </label>
+        <label htmlFor="date">
+          Date
+          <div className="date" id="date">
+            {format(new Date(schoolData.date.from), 'LLL y')} -{' '}
+            {format(new Date(schoolData.date.to), 'LLL y')}
+          </div>
+        </label>
+      </div>
+    );
+
+    if (isEditing) {
+      content = (
+        <div className={styles.infoContainer}>
+          <label htmlFor="school">
+            School
+            <input
```

Comment:
always declare input's `type` attribute. It gives more information about what is being edited and what to expect